sdk/schemaregistry/schema-registry-json/src/models.ts

```diff
@@ -51,3 +51,13 @@ export interface JsonSerializerOptions<MessageT> {
    */
   messageAdapter?: MessageAdapter<MessageT>;
 }
+
+/**
+ * The options to the deserialize method.
+ */
+export interface DeserializeOptions {
+  /**
+   * Validate the value against the schema
```

Comment:
should document what the return value means. Is `true` that it passes or that it fails validation?